### ISSUE TITLE
Coded anti-semitism in a stickerpack

### ISSUE THREAD

**Issue by celesteh** [signalstickers/signalstickers#848]
The [Mark Lizardberg sticker pack](https://signalstickers.com/pack/4685075ef688541d1f6ead50d79d047c)  uses coded anti-Semitic tropes, most obviously the lizard thing, but also implied devil horns in one of the images.  As Mark Zuckerberg actually is Jewish, this is especially inappropriate and the pack should be removed.

Edit to add: Context on the lizard people trope:
"Contemporary belief in reptilians is mostly linked to British conspiracy theorist David Icke . . .. his writings on the reptilian conspiracy theory are clearly evocative of the centuries-old blood-libel conspiracy theory, which alleged that a cabal of Jews were controlling the world and drinking the blood of Christian children."  https://www.businessinsider.in/thelife/news/the-bizarre-origins-of-the-lizard-people-conspiracy-theory-embraced-by-the-nashville-bomber-and-how-its-related-to-qanon/articleshow/80160505.cms

"The "lizard people" conspiracy theory in its own is deeply rooted in antisemitic tropes"  https://www.jpost.com/diaspora/antisemitism/the-umbrella-academy-under-fire-for-alleged-use-of-antisemitic-tropes-637558

https://www.theguardian.com/books/2001/mar/17/features.weekend

**Comment by romainricard** [comment]
Thanks for reporting this, @celesteh !

I didn't know about the roots of the lizard thing, and I've never heard of Icke. I did some research, and this man is clearly antisemite, no doubt about it.

However, I'm not sure of how it relates with this pack. Conspiracy theories have been parodied and made fun of, and some are now part of the popular culture, or at least the Internet culture. I think this pack follows this trends, and is _simply_ and _innocently_ an attempt to criticize Facebook, using [this now widespread meme](https://mashable.com/2018/04/05/mark-zuckerberg-memes/). Lots of people see Facebook (of which Zuckerberg is the key figure) as deeply evil, so if you need to make fun/criticize of Facebook, it seems logical to make fun of Zuckerberg.

I think/hope that Zuckerberg being Jewish has nothing to do with it.

I already have multiple reports about antisemitics/racist packs, and I've removed them, but in this case, I'll leave it.

**Comment by celesteh** [comment]
I appreciate the work you're doing and I know moderating is a thankless job. I do want to urge you to reconsider this case. 

My wife is Jewish and was very enthusiastically recommanding your sticker site to all of her friends (she loves the American Chopper one), but when she saw the Lizardberg one, she was visibly upset.  She views it as "unsubtle" and clearly antisemitic.

I'm a gentile, so I understand if you don't want to take my word for this, but perhaps you could ask a few Jews you know to take a look at and give their opinions.

I've got no love for Zuckerberg, but I do love my wife. Obviously, any member of a minority group needs a thick skin to go online, but it would be nice if this project went back to just being fun instead of communicating to her that she's not really part of the community. This is less visible than some of the stuff you've removed, but it's still visible to those it effects. It's definitely possible that the creator of this pack had no ill intentions and stumbled on an antisemitic trope unknowingly, alas.

Again I appreciate the work you've done. I'm sure you've seen some truly offensive stuff.

**Comment by Be-ing** [comment]
> simply and innocently an attempt to criticize Facebook

No, this is very clearly invoking antisemitic tropes. There are tons of ways to criticize Facebook without invoking antisemitic tropes.

**Comment by romainricard** [comment]
I talked with a lot of people, including Jewish friends, and got mixed opinions. Although I'm still not convinced that this pack is antisemitic -as I explained in my previous comment-, I decided to remove it. From my point of view, both keeping it online and removing it are bad solutions, but I'm not in the best position to judge if this pack is really antisemitic or not, and I want everybody to feel welcomed on signalstickers. Removed in https://github.com/signalstickers/stickers/commit/5e9c38513dfb2bc13f87b6a44adfc4157101eeeb
